ess/src/API/EMBC.Tests.Integration.ESS/Resources/EvacuationTests.cs

```diff
@@ -286,11 +282,7 @@ private EvacuationFile CreateTestFile(Evacuee primaryContact)
                     },
                     Type = NeedsAssessmentType.Preliminary,
                     Insurance = InsuranceOption.Yes,
-                    CanProvideClothing = true,
-                    CanProvideFood = true,
-                    CanProvideIncidentals = true,
-                    CanProvideLodging = true,
```

Comment:
same but not going to keep pointing it out

Reply:
same